src/Controller/TranslatorController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2020 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+namespace App\Controller;
+
+/**
+ * Translator controller.
+ *
+ * @property \App\Controller\Component\TranslatorComponent $Translator
+ */
+class TranslatorController extends AppController
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function initialize(): void
```

Comment:
add `@codeCoverageIgnore` or (better) add `@covers ::initialize()` to `TranslatorControllerTest::testTranslate()`